lib/grape/http/headers.rb

```diff
@@ -62,6 +62,13 @@ module Headers
       def self.find_supported_method(route_method)
         Grape::Http::Headers::SUPPORTED_METHODS.detect { |supported_method| supported_method.casecmp(route_method).zero? }
       end
+
+      def self.rackified_header(key)
```

Comment:
Name it `header_name`? But I think this method shouldn't exist. It's very much an internal thing of what the header is stored as. We have a `transform` method that should `downcase` it.